runtime/queries/sourcepawn/highlights.scm

```diff
@@ -0,0 +1,308 @@
+(identifier) @variable
+; Assume all-caps names are constants
+((identifier) @constant
+  (#lua-match? @constant "^[A-Z][A-Z0-9_]+$"))
```

Comment:
These two matches on `(identifier)` should be at the bottom of the file rather than the top. In neovim the precedence of the queries is the opposite of what it is in Helix (and tree-sitter-cli): in Helix the higher patterns in the file take precedence over the lower ones. So as-written these `(identifier)` matches will match before any of the patterns below that also match `(identifier)` like on L7 or L10. You will also want to swap the order of these two matches since the one with the `#match?` is more specific (so it should be higher precedence) than the one on L1.